### PR TITLE
Align schema with intake forms

### PR DESCRIPTION
**Pull request by cbell98**
## Description
https://www.loom.com/share/5b6653a269114c77a8610e8d4aebb2d5 

Removed "Do you live in the US" question from database schema.py and user_generators.py files.  Added "How did you hear about Underdog Devs?" field in Mentee class of database schema.py and user_generators.py. 

Fixes # (issue)

## Type of change

- [x] Bug fix

## Checklist:

- [x] My code follows PEP8 style guide
- [x] I have removed unnecessary print statements from my code
- [x] I have made corresponding changes to the documentation if necessary
- [x] My changes generate no errors
- [x] No commented-out code
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by AdamHVu — APPROVED** [review]
Very simple change, nothing wrong here!

**Review by tammer81 — APPROVED** [review]
This is a work of art.

File: data_generators/user_generators.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 
 class Printable:
+    "Creates string generated from dictionary"
```
[the remark this answers is not in the document]
Comment (cbell98) [reply data_generators/user_generators.py]: Changed this docstring to read "Parent Class"

**Review by Lopez-John — APPROVED** [review]
I don't see any conflicts between the schema.py file and the user_generator.py file after the chance. Great work.

**Review by GermanParra — APPROVED** [review]
Excellent work!

**Pull request merged**